libs/designsystem/src/lib/components/modal/alert/alert.component.ts

```diff
@@ -9,6 +16,7 @@ import { WindowRef } from '../../../types/window-ref';
   styleUrls: ['./alert.component.scss'],
   // tslint:disable-next-line: no-host-metadata-property
   host: { '[class.ion-page]': 'false' }, //Ensure ion-page class doesn't get applied by Ionic Modal Controller
+  changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
Do we know how onPush handles observables which the title input property can take?
```typescript
title$: Observable<string>;
  @Input()
  set title(title: string | Observable<string>) {
    this.title$ = typeof title === 'string' ? of(title) : title;
}
```

Reply:
This should be safe. Observables are comparable to Immutable inputs in this regard. See more here: https://vsavkin.com/change-detection-in-angular-2-4f216b855d4c